README.md

```diff
@@ -1,27 +1,32 @@
-# Project Name
+# Fcc Cat Photo App
 
-> One paragraph statement about the project.
+> A project using Basic HTML in styling of Cat Photos
 
-![screenshot](./app_screenshot.png)
+![cat photo](image.png)
 
 Additional description of the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 elements
+  - Ordered list <ol>,
+  - Unordered list <ul>
+  - Form, input, figure, main,
+- Linters
+  Prettier
+  Stylelint
+- Github Action
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+![cat photo app](<02-/Screenshot (177).png>)
```

Comment:
Let's use a deploy link here, we can add the vercel deploy link here instead of an image.

Reply:
okay sir